packages/common/src/types/schema.ts

```diff
@@ -1734,7 +1734,7 @@ export type ItemNode = {
   atcCategory: Scalars['String'];
   availableBatches: StockLineConnector;
   code: Scalars['String'];
-  ddd: Scalars['Float'];
+  ddd: Scalars['String'];
```

Comment:
this one was a surprise; have updated as required.

Reply:
what is "ddd"?

Reply:
Defined Daily Dose - which, according to WHO is "The assumed average maintenance dose per day for a drug used for its main indication in adults"